dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <Card class="col-12 page-wrapper-card p-3">
-    <template #title>Create EU Taxonomy Dataset for a Financial Company/Service</template>
+    <template #title>
+      <span data-test="pageWrapperTitle"
+        >{{ editMode ? "Edit" : "Create" }} EU Taxonomy Dataset for a Financial Company/Service</span
+      >
+    </template>
```

Comment:
Originally there was a review comment by me that said "rename the components to 'EuTaxo...' instead of 'EUTaxo...'"

We reverted the renaming because it confused git when merging main into this branch.

Now at the end of our development efforts:  Shouldn't we rename it back to "EuTaxo..." ?   Otherwise it breaks our naming pattern.

Reply:
I'd do the renames of files in a separate branch after the merge. Last Friday we again ran into issues that did cost us an hour or more since smoothScroll.ts was renamed to SmoothScroll.ts ...

Reply:
Ok.  I will resolve this conversation after a sub-task is created to reflect this ToDo of renaming it in a seperate branch.

Reply:
(same for ``updateObjectUtils`` etc.)

Reply:
https://jira.d-fine.dev/secure/RapidBoard.jspa?rapidView=246&projectKey=DALA&view=detail&selectedIssue=DALA-1591 Subtask has been created for this.